airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/copy/CopyConsumerFactory.java

```diff
@@ -148,6 +148,7 @@ private static void closeAsOneTransaction(final List<StreamCopier> streamCopiers
       for (final var copier : streamCopiers) {
         copier.removeFileAndDropTmpTable();
       }
+      db.close();
```

Comment:
@cgardens, @sherifnada, I found that if we don't close the database connection here, the Databricks destination acceptance test will stuck. This is a new issue. Previously it was not the case.

Do you know any recent change that may lead to this new issue? If not, the only alternative explanation I can think of is that we upgraded the Spark JDBC Driver. However, nothing in the release note seems related.